src/process.rs

```diff
@@ -112,41 +112,47 @@ pub fn process(
 
     let pid = proc.pid().unwrap(); // Must exist for a newly opened process
     thread::spawn(move || {
-        let mut comms = proc.communicate_start(None);
+        let mut comms = proc
+            .communicate_start(None)
+            .limit_time(Duration::from_micros(500));
+
+        let comm_data = {
+            match comms.read_string() {
+                Ok(data) => {
+                    // Just drop comms and give eof'd data
+                    (data.0, data.1)
+                }
```

Comment:
```suggestion
                // Just drop comms and give eof'd data
                Ok(data) => (data.0, data.1)
```
![image](https://user-images.githubusercontent.com/25136265/148287564-b912cb0a-1dad-4396-b87f-1c53f5ebbc2d.png)